etc/profile-m-z/rawtherapee.profile

```diff
@@ -0,0 +1,40 @@
+# Firejail profile for rawtherapee
+# Description: Free cross-platform raw image processing program
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include rawtherapee.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/RawTherapee
```

Comment:
Firejail's logic assumes a `noblacklist foo` is warranted when a `blacklist foo` exists somewhere in our other included files. So you'll need to add `blacklist ${HOME}/.cache/RawTherapee` and `blacklist ${HOME}/.config/RawTherapee` to `disable-programs.inc` to protect those paths from being accessible in other application sandboxes.

Reply:
Done. :+1: